bindings/ios/MEGASdk.mm

```diff
@@ -2145,6 +2146,8 @@ - (NSMutableArray *)recentActions {
         [recentActionBucketMutableArray addObject:recentActionBucket];
     }
     
+    delete megaRecentActionBucketList;
```

Comment:
wow, if have to manually deallocate the objects like this, there are more, at least the banner API binding, need to be updated as well.

Reply:
It will depend, you need to do it if the SDK saids: "* You take the ownership of the returned value."

Reply:
If the SDK documentation states that you take the ownership of the returned object, @joel-meng-nz , you do need to deallocate them, yes.

Is the SmartBanner feature included in 5.4? If not, please, fix your bindings in a different branch/PR. Thanks

Reply:
Navarro already told me it does not. Merging this PR...